types/src/front/assistant/actions/browse.ts

```diff
@@ -0,0 +1,46 @@
+import * as t from "io-ts";
+
+import { ModelId } from "../../../shared/model_id";
+import { BaseAction } from "../../lib/api/assistant/actions";
+
+export type BrowseConfigurationType = {
+  id: ModelId;
+  sId: string;
+  type: "browse_configuration";
+  name: string | null;
+  description: string | null;
+};
+
+export const BrowseResultSchema = t.type({
+  content: t.string,
+  response: t.type({
```

Comment:
as discussed [in thread](https://dust4ai.slack.com/archives/C050SM8NSPK/p1718191538274749) IMHO here we should have an optional error field (and null content if so)
ip & port are not necessary I think 
url should be the exact url requested as input (maybe it should be a map instead of an array)
if there's a redirect, this should be added somewhere in the return